docs/modules/concepts/pages/storage/users.adoc

```diff
@@ -0,0 +1,59 @@
+= User Directory
+:navtitle: Users
+
+== Overview
+
+Whereas xref:storage/mailbox.adoc[mailboxes] are clearly a core component of
```

Comment:
We might be defining more clearly what a user is.

I propose:

`As user is someone entitled to receive, store and retrieve email on this given server.`

Reply:
Ok. I will check the description of User in the "User Model" section, update if necessary, and link that that. Would that be ok?

Reply:
Done